apps/docs/pages/guides/getting-started/quickstarts/kotlin.mdx

```diff
@@ -0,0 +1,190 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import StepHikeCompact from '~/components/StepHikeCompact'
+
+export const meta = {
+  title: 'Use Supabase with Android Kotlin',
+  subtitle:
+    'Learn how to create a Supabase project, add some sample data to your database, and query the data from a Android Kotlin app.',
+  breadcrumb: 'Framework Quickstarts',
+}
+
+<StepHikeCompact>
+
+  <StepHikeCompact.Step step={1}>
+    <StepHikeCompact.Details title="Set up a Supabase project">
+
+    [Create a new project](https://app.supabase.com) in the Supabase Dashboard.
+
+    After your project is ready, create a table in your Supabase database using the [SQL Editor](https://app.supabase.com/project/_/sql) in the Dashboard. Use the following SQL statement to create all tables.
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+
+     ```sql SQL_EDITOR
+      -- Create products table
+      create table
+        public.products (
+          _id bigint generated by default as identity not null,
+          productid text not null,
+          name text null,
+          description text null,
+          price real null,
+          image text null,
+          category text null,
+          nutrition text null,
+          constraint products_pkey primary key (productid),
+        ) tablespace pg_default;
+
+      ````
+
+    </StepHikeCompact.Code>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={2}>
+
+    <StepHikeCompact.Details title="Create an Android app with Android Studio">
+      Setup environment for Android Development following official document.
+      Open Android Studio > New > New Android Project
+
+    </StepHikeCompact.Details>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={3}>
+
+    <StepHikeCompact.Details title="Install the Supabase client library">
+    Setup Supabase libraries requires some others libraries from ktor, so make sure all dependencies are added.
+    Replace place holder of the dependency versions `$supabase_version` and `$ktor_version` with latest versions as below
+    <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}> 
+      <a href="https://github.com/supabase-community/supabase-kt/releases" style={{ marginRight: '8px' }}> 
+          <img src="https://img.shields.io/github/release/supabase-community/supabase-kt?label=supabase-kt stable" alt="latest stable supabase-kt version"/> 
+      </a> 
+      <a href="https://github.com/supabase-community/supabase-kt/releases"> 
```

Comment:
```suggestion
      <a href="https://github.com/ktorio/ktor/releases"> 
```

Reply:
Btw, how do you find these badges straightforward? I tried searching but it took me lots of time and I had to replace some key here and there to update

Reply:
If you go to [shields.io](https://shields.io/) you can use the search bar on the top and enter terms like `github` or `maven` and you'll find the right one.

Reply:
Ah yeah thanks Jan. Another way is going to their official repository then copy 😅